scripts/build.js

```diff
@@ -40,5 +41,7 @@ compiler.run(function handler (err, stats) {
   process.stdout.write(output);
   process.stdout.write('\n\n');
 
-  ChromeExtensionUtils.pack(manifest);
+  ChromeExtensionUtils.pack(manifest, {
+    zip: isProd
```

Comment:
The Manifest Class doesn't have a `name` property right now. So we should whether add it, or pass the name as an option here